ert_storage/testing/testclient.py

```diff
@@ -200,16 +201,37 @@ def gql_execute(
         check: bool = True,
     ) -> dict:
         doc = self.gql_client.execute(request_string, variable_values=variable_values)
-        if check and "errors" in doc:
+        if self.checked and check and "errors" in doc:
             raise ClientError(f"GraphQL query returned an error:\n{pformat(doc)}")
 
         return doc
 
+    def _check(
+        self, check_status_code: Optional[int], response: requests.Response
+    ) -> None:
+        if (
+            not self.checked
```

Comment:
What's the purpose of `self.checked` ?

Reply:
Whether the TestClient should check and raise an exception. If the user doesn't want it (`self.checked is False`) then we don't perform a check.